CHANGELOG.md

```diff
@@ -1,8 +1,19 @@
 # Changelog
-## HEAD
-* Replace Base64 implementation with Java 8's #82
+## 1.14.0 -- 2019-10-10
+
+### Minor Changes
 * Add ExtraDataSupplier to Metastore #76
 * Add support for overriding KMS requests in DirectKMSMaterialProvider #76
+* Allow DoNotEncrypt and DoNotTouch to be used a a field level #95
```

Comment:
Typo “as” a field level

Reply:
Actually, should be "at". Fixing it and also bumping date to tomorrow.